File: solutions/easy/1295.Find_Numbers_with_Even_Number_of_Digits/1295.Find_Numbers_with_Even_Number_of_Digits.ts
function findNumbers(nums: number[]): number {
  let evenNumbers = 0;
  for(let i = 0; i < nums.length; i++) {
      let digits = 0;
      let n = nums[i];
      while (n > 0) {
          digits++;
          n = Math.floor(n / 10);          
      }
      if (digits % 2 === 0) evenNumbers++;
  }
  return evenNumbers;
};

File: solutions/easy/1295.Find_Numbers_with_Even_Number_of_Digits/1295.Find_Numbers_with_Even_Number_of_Digits.py
from ast import List
from math import floor


class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        evenAmountOfNums = 0
        for index in range(len(nums)):
            digits = 0
            n = nums[index]
            while n:
                digits += 1
                n = floor(n / 10)

            if digits % 2 == 0:
                evenAmountOfNums += 1
        return evenAmountOfNums
